fixtures/dom/src/components/fixtures/custom-elements/index.js

```diff
@@ -0,0 +1,47 @@
+import FixtureSet from '../../FixtureSet';
+import TestCase from '../../TestCase';
+
+const React = window.React;
+const ReactDOM = window.ReactDOM;
+
+class HelloWorld extends React.Component {
+  render() {
+    return <h1>Hello, world!</h1>;
+  }
+}
+
+// Babel breaks web components.
+// https://github.com/w3c/webcomponents/issues/587
+// eslint-disable-next-line no-new-func
+const MyElement = new Function(
+  'React',
+  'ReactDOM',
+  'HelloWorld',
+  `
+return class MyElement extends HTMLElement {
+  constructor() {
+    super();
+    const shadowRoot = this.attachShadow({ mode:'open' });
+    ReactDOM.render(React.createElement(HelloWorld), shadowRoot);
+  }
+}`
+)(React, ReactDOM, HelloWorld);
+
+customElements.define('my-element', MyElement);
```

Comment:
@gaearon this is causing the fixtures to fail in browsers that don't have `customElements`. We should either provide a polyfill or only run this fixture in browsers that support the custom element registry.